src/main/java/com/mathworks/ci/MatlabBuild.java

```diff
@@ -67,6 +70,8 @@ default void copyFileInWorkspace(String sourceFile, String targetFile, FilePath
         targetFilePath.copyFrom(in);
         // set executable permission
         targetFilePath.chmod(0755);
+     
+        

```

Comment:
Try to avoid unnecessary whitespace changes. They make it more difficult to see where the actual code changes are. If possible, try to get the review down to just the lines where there are actual code changes (unless the existing whitespace needs to be fixed).